From 2ab986e918449e2d13797c6e52088ca34b7e0a89 Mon Sep 17 00:00:00 2001
From: Jose Lopes <jabolopes@gmail.com>
Date: Fri, 19 Jul 2019 00:04:18 +0200
Subject: [PATCH 2/3] Don't generate /tftpboot in root directory of the FS.


diff --git a/Makefile b/Makefile
index 09944da..7b90807 100644
--- a/Makefile
+++ b/Makefile
@@ -421,8 +421,6 @@ ifeq ($(OSABL),YES)
 	cp -f $(RT28xx_DIR)/os/linux/$(MODULE)_apsta_util.ko /tftpboot
 	cp -f $(RT28xx_DIR)/os/linux/$(MODULE)_apsta_net.ko /tftpboot
 endif
-else
-	cp -f $(RT28xx_DIR)/os/linux/$(MODULE)_sta.ko /tftpboot 2>/dev/null || :
 ifeq ($(OSABL),YES)
 	cp -f $(RT28xx_DIR)/os/linux/$(MODULE)_sta_util.ko /tftpboot 2>/dev/null || :
 	cp -f $(RT28xx_DIR)/os/linux/$(MODULE)_sta_net.ko /tftpboot 2>/dev/null || :
-- 
2.14.5

